instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/internal/SemconvStability.java

```diff
@@ -51,18 +36,16 @@ public final class SemconvStability {
       }
     }
 
-    emitOldHttpSemconv = oldHttp;
-    emitStableHttpSemconv = stableHttp;
     emitOldJvmSemconv = oldJvm;
     emitStableJvmSemconv = stableJvm;
   }
 
   public static boolean emitOldHttpSemconv() {
-    return emitOldHttpSemconv;
+    return false;
   }
 
   public static boolean emitStableHttpSemconv() {
-    return emitStableHttpSemconv;
+    return true;
   }
```

Comment:
Is it a little silly now to even keep these here? It looks like the call sites have (mostly?) been removed, so I'm wondering what the motivation is in keeping them.

Reply:
There's tons of no-longer-needed code guarded by these, and I wanted to remove it in a separate PR. This one is already HUMONGOUS enough.

Reply:
Legit. I just didn't see it, so yeah, make sense to split it into smaller change units.